### ISSUE TITLE
Allow empty permissions

### ISSUE THREAD

**Issue by peaceiris** [rhysd/actionlint#170]
## Overview

We can set an empty `permissions` to disable permissions for all.
Can actionlint update the rule to allow the empty `permissions`?

## Current behavior

```console
$ actionlint --version
1.6.15
installed by building from source
built with go1.18.3 compiler for darwin/arm64
$ actionlint
"permissions" section should not be empty. please remove this section if it's unnecessary [syntax-check]
   |
   |     permissions: {}
```

## References

> https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
>
> You can use the following syntax to disable permissions for all of the available scopes:
> 
> ```yaml
> permissions: {}
> ```

**Comment by rhysd** [comment]
Yes, it shuould be ok to leave it empty.

**Comment by peaceiris** [comment]
I have made #171 ready for review.
